### ISSUE TITLE
[bug] model does not fit correctly when the image is not normalized

### ISSUE THREAD

**Issue by kunzaatko** [JuliaAstro/PSFModels.jl#11]
Hi! Like your work!
There is a slight problem when trying to fit a model to bead image in practice... The optimization without specifying the `amp` parameter does not work with real bead images, because they are usually not normalized to `(0,1)`. For example, I have this image of a bead:
![unnormalized_bead](https://user-images.githubusercontent.com/56647779/161416820-98208471-afee-4dd1-93d5-89d93e81ff8e.png)

and the fitted `gaussian` PSF model gives:
![unnormalized_fit](https://user-images.githubusercontent.com/56647779/161416831-b8be04ce-823b-45d0-9a4e-fbb14226957f.png)

When I normalize the bead image by
```julia
bead .-= minimum(bead)
bead ./= maximum(bead)
```
then the image looks like this:
![normalized_bead](https://user-images.githubusercontent.com/56647779/161416837-219731d9-7f93-4073-b7d9-9ac00cd6669e.png)

and the fitted `gaussian` PSF model looks like this:
![normalized_fit](https://user-images.githubusercontent.com/56647779/161416839-82b4550b-29a5-4d87-aba3-c9c9da98244b.png)

Which is a much better fit.
Note, that just specifying the `amp = 1` parameter would not help in this case, because it makes the optimization much more likely to land at a local optimum. This could be solved only by setting the `amp` parameter very close to the true value. Here is the result when optimizing against `amp` also with the same bead:
![unnormalized_fit_with_amp](https://user-images.githubusercontent.com/56647779/161416843-7b364ca6-6104-463d-acbf-c77eff8d053d.png)

**Comment by kunzaatko** [comment]
The best way to approach this would probably be to predefine the `amp` before the fitting to `maximum(im)` or to fit a normalized image.

**Comment by kunzaatko** [comment]
Alternatively, it could be only noted in the docs so that users are cautious
.

**Comment by mileslucas** [comment]
I'm not sure this is a bug, like you said if you want to fit the amplitude, you need to specify it, otherwise you could do

```julia
func_kwargs = (;amps=fixed_amp_value)
PSFModels.fit(...; func_kwargs=func_kwargs)
```
> Alternatively, it could be only noted in the docs so that users are cautious

PRs are welcome! As a note, by the way, all of the astropy PSF models require an amplitude, and don't make any assumptions about normalized or unnormalized data, the same as we do here. Are there other tools you are used to that "automatically" apply these pre-processing steps? If, so, I'd be interested in checking them out!

**Comment by kunzaatko** [comment]
Right.., I think _bug_ was not the right word. It is just that the _'default'_ method for fitting does not adhere to the probably most frequent use of it... That is that fitting the normalized PSF model into a most surely unnormalized view into the image. Right? I will go through the docs and add some comment about this. With that, I think that it would be settled. Would that be OK?
